docusaurus.config.js

```diff
@@ -1,58 +1,17 @@
 const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 /* custom */
-const CopyRight = `
-<div class="footer-box">
-	<div class="footer-item">
-		<a href="https://www.apachecon.com/acna2022/">
-			<img title="ApacheCon North America" alt="ApacheCon North America" src="https://www.apachecon.com/event-images/acna2022-wide-dark.png" class="footer-pc-img" style="max-width: 310px;">
-		</a>
-		<a href="https://www.apachecon.com/acna2022/">
-			<img title="ApacheCon North America" alt="ApacheCon North America" src="https://www.apachecon.com/event-images/acna2022-wide-dark.png" class="footer-mobile-img" style="max-width: 114px;">
-		</a>
-		<p>
-			The Official Global Conference Series of The Apache Software Foundation,
-			connecting open source technology enthusiasts from around the world to
-			share the latest technology developments and practices.
-		</p>
-	</div>
-	<div class="footer-item">
-		<img class="footer-img" src="/image/incubator_feather_egg_gray_logo.png">
-		<p>
-			<span class="footer-sep">
-				Apache StreamPark is an effort undergoing Incubation at The Apache Software
-				Foundation (ASF), sponsored by the Incubator. Incubation is required of
-				all newly accepted projects until a further review indicates that the infrastructure,
-				communications, and decision making process have stabilized in a manner
-				consistent with other successful ASF projects. While incubation status
-				is not necessarily a reflection of the completeness or stability of the
-				code, it does indicate that the project has yet to be fully endorsed by
-				the ASF.
-			</span>
-		</p>
-	</div>
-	<div class="footer-copyright">
-		<div class="footer-block"></div>
-		<div class="footer-copyright-content">
-			<span>
-			        Copyright © 2019-${new Date().getFullYear()} The Apache Software Foundation.
-				Apache StreamPark, StreamPark, Apache, the Apache Feather logo, and the Apache Incubator project logo
-				are trademarks of The Apache Software Foundation
-			</span>
-		</div>
-	</div>
-</div>
-`
+
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-	title: 'StreamPark',
-	tagline: 'StreamPark',
-	url: 'https://streamxhub.com',
+	title: 'Apache StreamPark (Incubating)',
+	tagline: 'Apache StreamPark is a Extremely fast stream processing development framework, supported unified batch & streaming and data lake & data warehouse cloud-native bigdata platform, one-stop real-time computing cloud-native platform',
+	url: 'https://streampark.apache.org/',
 	baseUrl: '/',
 	onBrokenLinks: 'ignore',
 	onBrokenMarkdownLinks: 'warn',
 	favicon: 'image/favicon.ico',
-	organizationName: 'Streamxhub',
-	projectName: 'StreamPark',
+	organizationName: 'Apache',
+	projectName: 'Apache StreamPark',
```

Comment:
These lines are for deployment and they should be "apache" and "incubator-streampark-website". Since we don't use the `deploy` target, I suggest we simply remove them.
```suggestion
```

Reply:
Thank you for your reminder, Done